Formula/ffmpeg.rb

```diff
@@ -83,6 +84,7 @@ def install
       --enable-libmp3lame
       --enable-libopus
       --enable-librav1e
+      --enable-librist
```

Comment:
What does enabling this feature do? Would most users find it useful? If not, this feature is probably better enabled (possibly optionally) in an external tap like https://github.com/homebrew-ffmpeg/homebrew-ffmpeg.

Reply:
Actually the same reason as why SRT is standard in the list. We see that streaming over the open internet is getting bigger and bigger. At the moment SRT is the best known alternative, as it comes by default in Homebrew and some static builds, although in FFmpeg it is disabled by default in configure (https://hub.com/FFmpeg/FFmpeg/blob/master/configure ). SRT is developed and managed by single commercial company. RIST, on the other hand, was the collective design work of a large group of experts (companies) who have been providing parcel recovery services for many years. From the start, rist has been based on clear and open standards. RIST would be a good addition to provide more alternative solutions for streaming over the open internet.

Reply:
I don't think that answers either of my questions. What does enabling this do? Why would most users find it useful?

Reply:
``--enable-librist`` enabled RIST functionally in FFmpeg, otherwise you can't use RIST. My main reason is that if we do keep SRT standard in homebrew and "hide" other alternative solutions. Users then have to take an extra step to use an alternative solution. As a result, you do not get a fair distribution between alternatives. In addition, there are now products on the market that support RIST, and now you can't communicate with them with the standard homebrew solution.

Reply:
>`--enable-librist` enabled RIST functionally in FFmpeg, otherwise you can't use RIST.

This is pretty obvious. Suppose I were an `ffmpeg` user, but I don't know what `librist` is. Why would I want it enabled with the accompanying dependency?

I'm not sure what your notion of "fair" here is, but I would guess it is also pretty unfair for users who won't use `librist` with `ffmpeg` to be forced to install `librist` anyway.

Reply:
> This is pretty obvious. Suppose I were an `ffmpeg` user, but I don't know what `librist` is. Why would I want it enabled with the accompanying dependency?

It allows users to send and receive audiovisual streams using the RIST protocol, similarly to SRT. It is a means to distribute mainly LIVE streams.

> I'm not sure what your notion of "fair" here is, but I would guess it is also pretty unfair for users who won't use `librist` with `ffmpeg` to be forced to install `librist` anyway.

RIST is likely to be a increasingly widely accepted alternative against SRT, that is fully based on international open standards. Most users are not compiling their own binaries and thus having this library dependency enabled with FFmpeg gives the user more choice to use this protocol.

RIST and SRT protocol allow to send live streams over open internet more reliably and with less jitter/latency than older protocols like RTSP/RTMP. Only allowing SRT by default in homebrew creates unfair competition for alternative solutions like RIST.

Reply:
Is it enabled in the binaries that the FFMPEG project distributes?

Reply:
SRT and RIST are default disabled in FFmpeg builds, also in the configure. You need manually enabled this features.

Reply:
Why don't they enable this by default? That seems like a reason to disable both rather than enable either.

Reply:
Because the default build of FFmpeg doesn't want to be too dependent on third parties. Compiling cost a lot of time. But I don't understand why SRT was added to the list, even though it was disabled by default? What are the rules for this type of liberties?

Reply:
It was enabled here because some users found it useful: #53837

How widely adopted is RIST?

Reply:
I have looked at the pull request and this pull request is by one of the developers of SRT.  I am independent and would like to see both protocols back in homebrew, because both protocols works similar of each other.  Only the vision of the organization is different.  More than 100 companies are currently working on RIST: https://www.rist.tv/news/2020/9/22/rist-forum-membership-surpasses-100-members

Reply:
Ok. That seems good enough. `librist` itself is also a fairly lightweight dependency, and (at the time of writing) has no other dependents.